docs-2.0/nebula-analytics.md

```diff
@@ -37,6 +37,8 @@ Nebula Analytics supports the following graph algorithms.
 |  PageRank             | It is used to rank web pages.          | Node importance measurement   |
 |  KCore                | k-Cores              | Node importance measurement   |
 |  DegreeCentrality     | It is a simple count of the total number of connections linked to a vertex.           | Node importance measurement   |
+|  DegreeWithTime        | Neighbor statistics based on the time range of edge ranks | Node importance measurement  |
+|  PageRankLDBC          | Rank web pages compatible with LDBC   | Node importance measurement  |
```

Comment:
what is PageRankLDBC? This is not a standard page rank algorithm.
It is temporary? or use URL link to explain the algorithm.